jupyterlab/runlab.sh

```diff
@@ -192,34 +217,37 @@ else
 fi
 # The Rubin Lap App plus our environment should get the right hub settings
 # This will need to change for JL 3
-cmd="jupyter-rubinlab \
+cmd="jupyter labhub \
      --ip='*' \
      --port=8888 \
      --no-browser \
      --notebook-dir=${HOME} \
-     --LabApp.shutdown_no_activity_timeout=43200 \
+     --hub-prefix='/nb/hub' \
```

Comment:
I thought frossie wanted this to be just `/nb`

Reply:
The _ingress_ is just "/nb".  For this, which is behind the configurable hub proxy, to find the hub, it needs to go to the hub route explicitly.